modules/nw-metallb-configure-bfdprofle.adoc

```diff
@@ -23,7 +23,7 @@ apiVersion: metallb.io/v1beta1
 kind: BFDProfile
 metadata:
   name: doc-example-bfd-profile-full
-  namespace: metallb-system
+  namespace: openshift-operators
```

Comment:
Noticed this by chance but the current (4.10) docs has more realistic values: https://docs.openshift.com/container-platform/4.10/networking/metallb/metallb-configure-bfd-profiles.html
It's a fix that was probably not ported on master?
We shouldn't get back on that